tenable/io/v3/vm/exclusions/api.py

```diff
@@ -0,0 +1,443 @@
+'''
+Exclusions
+==========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`exclusions <exclusions>`
+API endpoints.
+
+Methods available on ``tio.v3.vm.exclusions``:
+
+.. rst-class:: hide-signature
+.. autoclass:: ExclusionsAPI
+    :members:
+'''
+
+from datetime import datetime
+from typing import BinaryIO, Dict, List, Optional, Union
+from uuid import UUID
+
+from requests import Response
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.vm.exclusions.iterator import (ExclusionCSVIterator,
+                                                  ExclusionSearchIterator)
+from tenable.io.v3.vm.exclusions.schema import ExclusionSchema
+
+
+class ExclusionsAPI(ExploreBaseEndpoint):
+    '''
+    This will contain all methods related to exclusions
+    '''
+    _path: str = 'api/v3/exclusions'
+    _conv_json: bool = True
+    _schema = ExclusionSchema()
+
+    def create(self,
+               name: str,
+               members: List,
+               start_time: Optional[str] = None,
+               end_time: Optional[str] = None,
+               description: Optional[str] = None,
+               frequency: Optional[str] = None,
+               interval: Optional[int] = None,
+               weekdays: Optional[List] = None,
+               day_of_month: Optional[int] = None,
+               enabled: Optional[bool] = True,
+               network_id: Optional[UUID] = None
+               ) -> Dict:
+        '''
+        Create a scan target exclusion.
+
+        :devportal:`exclusions: create <exclusions-create>`
+
+        Args:
+            name (str): The name of the exclusion to create.
+            members (list):
+                The exclusions members.  Each member should be a string with
+                either a FQDN, IP Address, IP Range, or CIDR.
+            start_time (datetime): When the exclusion should start.
```

Comment:
(datetime, optional)?

Reply:
this field is required in request data as per the API behavior.